client/flutter/lib/pages/news_feed.dart

```diff
@@ -18,28 +16,32 @@ class NewsFeed extends StatelessWidget {
                 "Situation reports provide the latest updates on the novel coronavirus outbreak.",
             imageProvider:
                 AssetImage("assets/news_press/news_situation_reports.png"),
-            url: "https://who.int",
+            url:
```

Comment:
Where are these URLs from? do we have a spreadsheet with them yet?

Reply:
The accompanying WhatsApp WHO chatbot. Check v1 Explorations in Figma to see the chatbot content and the app UI side by side.

Reply:
they're also listed here https://github.com/WorldHealthOrganization/app/issues/586